components/tabs/style/index.less

```diff
@@ -364,6 +364,10 @@
     .@{tab-prefix-cls}-tab {
       text-align: right;
     }
+    .@{tab-prefix-cls}-tab-active {
+      font-weight: normal;
+      text-shadow: 0px 0px 1px @tabs-card-active-color;
```

Comment:
Thanks, I don't think `text-shadow` is a good idea, and it's doesn't support `IE 9` browser, I will close this PR, via https://github.com/react-component/tabs/pull/221#issuecomment-541290983

![image](https://user-images.githubusercontent.com/14831261/68657868-6ad3de00-056f-11ea-9246-6df7332fc633.png)